src/ui/EditTask.scss

```diff
@@ -1,5 +1,5 @@
 .tasks-modal {
-    > section {
+    section:not(:first-child) {
```

Comment:
This is probably worth adding a comment to say why... It sounds like it may be quite specific to field order...

Reply:
Oh, right. I rewrote this and so it is self documenting `section + section {}` means something specific to between `<section>` elements =)